app/scraper/loader.py

```diff
@@ -0,0 +1,54 @@
+from json import loads, JSONDecodeError
+
+from urllib.parse import urlencode
+from tornado.options import options
+from tornado.httpclient import AsyncHTTPClient
+
+from app.scraper.utils import prepare_post_data, prepare_request
+
+
+__all__ = ["load_page", "load_schedule", "load_teachers_or_groups", ]
+
+
+async def load_page(**kwargs):
+    # Todo: provide docstring
+    request = prepare_request(**kwargs)
+    response = await AsyncHTTPClient().fetch(request=request)
+    return response.body.decode(options.base_encoding)
+
+
+async def load_schedule(**kwargs):
+    # Todo: provide docstring
+    post_data = prepare_post_data(**kwargs)
+    body = urlencode(post_data)
+
+    return await load_page(method='POST', body=body)
+
+
+async def load_teachers_or_groups(query='', faculty='0', teachers=False):
+    # Todo: provide docstring
+    if teachers:
+        api_code = options.teachers_api_code
+    else:
+        api_code = options.groups_api_code
+
```

Comment:
Can be replaced with
```python
api_code = options.teachers_api_code if teachers else options.groups_api_code
```

Reply:
Looks like this line is longer than 79 characters.

Reply:
Yep, leave it as it was